enterprise/cmd/repo-updater/internal/authz/perms_syncer.go

```diff
@@ -746,6 +746,13 @@ func (s *PermsSyncer) syncRepoPerms(ctx context.Context, repoID api.RepoID, noPe
 		// Skip repo if unimplemented
 		if errors.Is(err, &authz.ErrUnimplemented{}) {
 			log15.Debug("PermsSyncer.syncRepoPerms.unimplemented", "repoID", repo.ID, "err", err)
+
+			// We should still touch the repo perms so that we don't keep scheduling the repo
+			// for permissions syncs on a tight interval.
+			if err = s.permsStore.TouchRepoPermissions(ctx, int32(repoID)); err != nil {
+				log15.Warn("Error touching permissions for UnImplemented authz provider", "err", err)
```

Comment:
Nittiest of nitty nits. :stuck_out_tongue: 

```suggestion
				log15.Warn("Error touching permissions for Unimplemented authz provider", "err", err)
```